packages/cli/src/index.ts

```diff
@@ -77,14 +77,14 @@ async function main(
   const transformTask = async (transform: TransformConfig) => {
     if (transform.mode === 'ts-implicit') {
       const transformer = new TypedSqlTagTransformer(pool, config, transform);
-      return transformer.start(isWatchMode);
+      return transformer.start(isWatchMode, fileOverride);
```

Comment:
Tested this and this change actually breaks b/c this transformer doesn't support fileOverride.

Also, we should make the globSync take `absolute: true` because right now it confusingly matches against the `{srcDir}/file/path/filename.sql`. And update the `--help`.

Anyway, since there seems like there isn't really movement on these PRs, going to abandon this and maybe someone can pick it up if things change with the project.